subsys/bluetooth/controller/ll_sw/isoal.c

```diff
@@ -691,17 +720,27 @@ static isoal_status_t isoal_rx_framed_consume(struct isoal_sink *sink,
 		 *   0b10  Part(s) of the SDU were not received correctly. This is reported as
 		 *         "lost data".
 		 */
-		if (pdu_err) {
-			sp->sdu_status |= meta->status;
-		} else if (seq_err) {
+		if (seq_err) {
 			sp->sdu_status |= ISOAL_SDU_STATUS_LOST_DATA;
+		} else if (pdu_err) {
+			sp->sdu_status |= meta->status;
+		}
+
+		if (sp->fsm == ISOAL_START) {
+			/* Sequence number should be incremented if an error
+			 * occurs at the beginning.
+			 */
+			session->seqn++;
```

Comment:
Why?

Reply:
@niag-Oticon, care to explain?

Reply:
It's because if there is a error in reception, an SDU will be released at line 738. If the sequence number is not incremented then we will be pushing SDUs to the host with reused packet sequence numbers whenever an error occurs. This sequence number will end up being the value in the ISO_Data_Load field Packet_Sequence_Number, so it should not repeat especially between error and good SDUs.

Reply:
That makes sense (in fact the sequence number shall be incremented per SDU interval, even if no SDUs are sent by the remote device). 

I feel like the comment is misleading then, and perhaps something like 
```c
			/* Sequence number shall be incremented even if an error
			 * occurs at the beginning.
			 */
```